backend/.env.example

```
@@ -79,3 +79,18 @@ OPENSEARCH_USERNAME=admin
+AUTH_PUBLIC_KEY_PATH=public.pem
+CERT_PATH_PUBLIC=public.pem
+
+# Voila Proxy Configuration
+VOILA_BASE_URL=http://voila:8866
+
+# Gold Lakehouse API
+# GOLD_LAKEHOUSE_API_URL=http://gold-lakehouse:8000
+GOLD_LAKEHOUSE_API_URL=http://10.10.80.19:8282
+GOLD_LAKEHOUSE_TIMEOUT=30000
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)